stacks/spica/packages/common/src/input/components/number-schema/number-schema.component.html

```diff
@@ -12,7 +12,7 @@
 <div class="description mat-hint" *ngIf="schema.description">
   If you don't want to limit values, you can keep the above lines empty.
 </div>
-<mat-slide-toggle #isEnum [checked]="schema.enum?.length > 0"
+<mat-slide-toggle [(ngModel)]="checked" #isEnum (change)="changed()"
```

Comment:
We don't need any additional properties since we can obtain if whether this schema have enums by checking the length of the enum property. 

So please rather use something like this 
`[ngModel]="this.schema.enum.length > 0"  (ngModelChange)="$event ? this.schema.enum = [] : removeEnum()" `

More readable and reactive.